overlord/devicestate/handlers_test.go

```diff
@@ -487,24 +487,23 @@ apps:
 	})
 }
 
-// TODO: rename preesed mode to just preseeding as much as possible,
-// preseed mode souns like a UC20 system mode but is just a snapd mode
-// but preseed snapd mode is a mouthful
-type preseedModeSuite struct {
-	preseedBaseSuite
+type preseedingClassicSuite struct {
+	preseedingBaseSuite
 }
 
-var _ = Suite(&preseedModeSuite{})
+var _ = Suite(&preseedingClassicSuite{})
 
-func (s *preseedModeSuite) SetUpTest(c *C) {
-	s.preseedBaseSuite.SetUpTest(c, true)
+func (s *preseedingClassicSuite) SetUpTest(c *C) {
+	classic := true
+	preseed := true
+	s.preseedingBaseSuite.SetUpTest(c, preseed, classic)
 }
 
-func (s *preseedModeSuite) TearDownTest(c *C) {
-	s.preseedBaseSuite.TearDownTest(c)
+func (s *preseedingClassicSuite) TearDownTest(c *C) {
```

Comment:
this whole method shouldn't be needed as just calls TearDownTest on the included struct